oracle/CHANGELOG.md

```diff
@@ -2,193 +2,352 @@
 
-* [Added] Upgrade dependencies for Python 3.7 binary wheels. See [#4030](https://github.com/DataDog/integrations-core/pull/4030).
+***Added***: 
+
+* Upgrade dependencies for Python 3.7 binary wheels. See [#4030](https://github.com/DataDog/integrations-core/pull/4030).
+
 
 ## 1.8.0 / 2019-06-01 / Agent 6.12.0
 
-* [Added] Support multiple results in custom queries. See [#3765](https://github.com/DataDog/integrations-core/pull/3765).
+***Added***: 
+
+* Support multiple results in custom queries. See [#3765](https://github.com/DataDog/integrations-core/pull/3765).
+
 
 ## 1.7.0 / 2019-05-14
 
-* [Added] Turn an info log into debug. See [#3661](https://github.com/DataDog/integrations-core/pull/3661).
-* [Added] Adhere to code style. See [#3552](https://github.com/DataDog/integrations-core/pull/3552).
+***Added***: 
+
+* Turn an info log into debug. See [#3661](https://github.com/DataDog/integrations-core/pull/3661).
+* Adhere to code style. See [#3552](https://github.com/DataDog/integrations-core/pull/3552).
+
 
 ## 1.6.0 / 2019-03-29 / Agent 6.11.0
 
-* [Added] Add custom_queries config globally. See [#3231](https://github.com/DataDog/integrations-core/pull/3231).
+***Added***: 
+
+* Add custom_queries config globally. See [#3231](https://github.com/DataDog/integrations-core/pull/3231).
+
 
 ## 1.5.0 / 2019-02-18 / Agent 6.10.0
 
-* [Added] Support Python 3. See [#3037](https://github.com/DataDog/integrations-core/pull/3037).
-* [Fixed] Fix tablespace metrics. See [#2841](https://github.com/DataDog/integrations-core/pull/2841).
+***Added***: 
+
+* Support Python 3. See [#3037](https://github.com/DataDog/integrations-core/pull/3037).
+
+***Fixed***: 
+
+* Fix tablespace metrics. See [#2841](https://github.com/DataDog/integrations-core/pull/2841).
+
 
 ## 1.4.0 / 2018-09-04 / Agent 6.5.0
 
-* [Added] Add process metrics. See [#1856](https://github.com/DataDog/integrations-core/pull/1856).
-* [Fixed] Add data files to the wheel package. See [#1727](https://github.com/DataDog/integrations-core/pull/1727).
+***Added***: 
+
+* Add process metrics. See [#1856](https://github.com/DataDog/integrations-core/pull/1856).
+
+***Fixed***: 
+
+* Add data files to the wheel package. See [#1727](https://github.com/DataDog/integrations-core/pull/1727).
+
 
 ## 1.3.0 / 2018-06-07
 
-* [Added] Support custom queries. See [#1528](https://github.com/DataDog/integrations-core/pull/1528).
-* [Added] [oracle]  Add ability to use the JDBC Driver instead of cx_Oracle. See [#1459](https://github.com/DataDog/integrations-core/pull/1459).
-* [FEATURE] ability to use the JDBC Driver instead of `cx_Oracle` to connect to the database. See [#1459](https://github.com/DataDog/integrations-core/issues/1459)
+***Added***: 
+
+* Support custom queries. See [#1528](https://github.com/DataDog/integrations-core/pull/1528).
+* [oracle]  Add ability to use the JDBC Driver instead of cx_Oracle. See [#1459](https://github.com/DataDog/integrations-core/pull/1459).
```

Comment:
nit:

```suggestion
* Add ability to use the JDBC Driver instead of cx_Oracle. See [#1459](https://github.com/DataDog/integrations-core/pull/1459).
```